src/components/core/styles/Checkbox.module.css

```diff
@@ -1,3 +1,12 @@
 .checkbox {
     margin: var(--spacers-dp12) 0 var(--spacers-dp8);
 }
+
+.checkbox label {
```

Comment:
Did you confirm that these styles don't negatively impact other places checkbox is used?

Reply:
Yes, I've checked all the layer dialogs and checkboxes look the same.